src/runtime/storageNG/tests/handle-test.ts

```diff
@@ -376,3 +397,167 @@ describe('SingletonHandle', async () => {
     }
   });
 });
+
+describe('EntityHandle', async () => {
+  it('can sync when the entity has both singleton and collection fields', async () => {
+    const manifest = await Manifest.parse(`
+      schema Simple
+        txt: Text
+        flag: Boolean
+        nums: [Number]
+    `);
+    const simpleSchema = manifest.schemas.Simple;
+    const simpleEntityClass = Entity.createEntityClass(simpleSchema, null);
+    const simpleEntity = new simpleEntityClass({txt: 'Text', flag: true, nums: [1, 2]});
+    const simpleMuxId = 'simpleMuxId';
```

Comment:
from the perspective of the Entity, the muxId is just the id, maybe rename this variable to simpleId?